# Current timestamp header
time stamp

# Left encoder ticks
int32 left_ticks

# Right encoder ticks
int32 right_ticks
